### ISSUE TITLE
Models view in dashboard throwing error

### ISSUE THREAD

**Issue by tylerdavis** [postgresml/postgresml#142]
When visiting the following URL `http://localhost:8000/models/81` I am receiving the following error.  This is happening on all models show pages.

```
KeyError at /models/81
KeyError at 'image_p50'

-- | --
http://localhost:8000/models/81
4.0.4
KeyError
'image_p50'
/app/app/views/models.py, line 35, in <dictcomp>
/usr/local/bin/python3
3.10.4
['/app',  '/usr/local/lib/python310.zip',  '/usr/local/lib/python3.10',  '/usr/local/lib/python3.10/lib-dynload',  '/usr/local/lib/python3.10/site-packages',  '/app/..']
```

**Comment by montanalow** [comment]
What Postgres type is the `image` column in your dataset? This looks like a hyperparam search with svm. Is this per chance on the iris dataset?

**Comment by tylerdavis** [comment]
It looks like the `Handwritten Digit Image Classifier` project.  The image column is `smallint[]`

**Comment by montanalow** [comment]
I'll release a new version with this fix as soon as I figure out what's up with the other issue you raised.

**Comment by tylerdavis** [comment]
Thanks @montanalow!

**Comment by montanalow** [comment]
I've released 0.9.2 with the fix
